### DISCUSSION TITLE
pmtiles with maplibre-gl-js globe?

### DISCUSSION THREAD

**Discussion by resiliencetheatre** [protomaps/PMTiles#492] [General]
Have anyone tried maplibre-gl-js upcoming globe support with pmtiles? I got some very broken tiles.

**Comment by bdon** [comment]
There should be no difference in any part of the code related to rendering globes, pmtiles only concerns the loading of the data.

**Reply by resiliencetheatre** [reply]
Excellent. Then it must be my code alone. Have you tried this and possible any example available?

**Reply by wipfli** [reply]
- Take your current MapLibre GL JS code which works with your PMTiles
- Update to MapLibre GL JS v5.0.0-pre.6
- Add `map.addControl(new maplibregl.GlobeControl(), 'top-right');`

Now you should have a little globe button on your map to toggle globe on and off.
